<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindow</class>
 <widget class="QMainWindow" name="SearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-2</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>571</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
border: 1px solid black;
border-radius: 50%;</string>
    </property>
    <property name="placeholderText">
     <string>Search...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(95, 95, 95);</string>
    </property>
    <item>
     <property name="text">
      <string>Artist</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Title</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Album</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #000000;
background-color: rgb(161, 161, 161);</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(161,161,161);color:white;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>100</y>
      <width>741</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>739</width>
       <height>379</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
